etcdserver/etcdserverpb/rpc.proto

```diff
@@ -331,8 +331,10 @@ message HashRequest {
 
 message HashResponse {
   ResponseHeader header = 1;
-  // hash is the hash value computed from the responding member's key-value store.
+  // Hash is the hash value computed from the responding member's key-value store.
   uint32 hash = 2;
+  // Revision is the key-value store revision at the moment hash API return. 
+  int64 revision = 3;
```

Comment:
we do not need this. we already have revision in the header. we just need to fill in the current revision at this time.